src/main/java/kitchenpos/products/tobe/domain/Product.java

```diff
@@ -0,0 +1,40 @@
+package kitchenpos.products.tobe.domain;
+
+import javax.persistence.*;
+import java.math.BigDecimal;
+
+@Entity
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    @Embedded
+    private ProductPrice price;
+
+    public Product() {
+    }
+
+    public Product(Long id, String name, BigDecimal price) {
+        this.id = id;
+        this.name = name;
+        this.price = new ProductPrice(price);
+    }
+
+    public Product(String name, BigDecimal price) {
+        this.name = name;
+        this.price = new ProductPrice(price);
```

Comment:
자기 생성자 호출하는 `this()`를 사용하면 어떨까요?

```suggestion
        this(null, name, price);
```